l10n_us_check_writing_address/models/res_partner.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.multi
+    def _display_address(self, without_company=False):
```

Comment:
I'm not sure how this is being used- but why not just use the built-in address widget?

Reply:
@lasley To avoid empty lines if there is no data, I think.